### PR TITLE
add timezone to the user profile 

### PR DESCRIPTION
**Pull request by mnglkhn**
The user is able to select a timezone to his/her profile.
This is useful when users are on different timezone and need to have date/time info displayed in their timezone.

### CODE REVIEW

File: Module.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Module extends \yii\base\Module
     /**
      * @var bool If true, users are required to enter a username
      */
-    public $requireUsername = false;
+    public $requireUsername = true;
```
Comment (codyayers) [inline Module.php]: This probably slipped into your commit, I don't think the author will want to default to a required username.

**Comment by amnah** [comment]
This is awesome! I'll look into getting it in tonight, thanks so much!

**Comment by mnglkhn** [comment]
I just realized that it is a patch to the ^3.0.0 branch
But the changes can be easily brought forward to the current branch

**Comment by mnglkhn** [comment]
closed by mistake; please feel free to reopen

**Comment by amnah** [comment]
Just added this in manually. Thanks again!!

**Comment by mnglkhn** [comment]
the dropdown list is rather long and it would be nice to use Kartik's
Select2 widget.
Not sure if you want to introduce a dependency but maybe a short
readme/hint would also help.

**Comment by amnah** [comment]
Ya, I don't really want to add anymore to this - my goal is to keep this module simple and basic. Note/comment added da5fa76

**Pull request closed without merging**